armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -857,9 +855,18 @@ def setUp(self):
         self.a = buildTestAssemblyWithFakeMaterial(name="HT9")
 
     def test_getSolidComponents(self):
+        """Show that getSolidComponents produces a list of solids, and is consistent with iterSolidComponents."""
         for b in self.a:
-            for c in getSolidComponents(b):
+            solids = getSolidComponents(b)
+            ids = set(map(id, solids))
```

Comment:
oh man... `map`s.... to the google lol

Reply:
read this as `ids = {id(s) for s in solids}`